dockerfiles/samtools-cloud/Dockerfile

```diff
@@ -15,17 +15,16 @@ ARG GCLOUD_SDK_TRANSIENT_DEP="curl gpg-agent gnupg python-pip python-setuptools
 ARG GCLOUD_SDK_DEPENDENCIES="lsb-release ${GCLOUD_SDK_TRANSIENT_DEP}"
 ARG DEBIAN_FRONTEND=noninteractive
 RUN apt-get -qqy update --fix-missing && \
-    apt-get -qqy dist-upgrade && \
-    apt-get -qqy install --no-install-recommends \
+    apt-get -qqy install --no-upgrade --no-install-recommends \
```

Comment:
I'm skipping upgrades for apt-get, which will only affect base packages for the docker os, not the installed conda packages. I am skipping the updates because they are done in sv-base-mini (so it's possible to fully update the os) however, if they are done independently in dependent docker images, they blow up the image size (the derived images have to store the diffs: removed packages + replaced new packages).